Project.toml

```diff
@@ -98,6 +104,7 @@ Libdl = "1.10"
 MacroTools = "0.5.13"
 Markdown = "1.10"
 Ncurses_jll = "6.4.1"
+Nemo = "0.50.0, 0.51.0"
```

Comment:
Drawback: with this one more package needs to be updated each time we make a breaking Nemo release :-(

Reply:
sure, it wasn't really an objection anyway, just an observation